src/components/App.js

```diff
@@ -135,19 +132,29 @@ const App = React.createClass({
         endPanel: SecondaryPanes({ horizontal }),
         endPanelCollapsed,
       }));
-  },
+  }
 
   render() {
     return this.state.horizontal ?
       this.renderHorizontalLayout() : this.renderVerticalLayout();
   }
-});
+}
+
+App.propTypes = {
+  sources: PropTypes.object,
+  selectSource: PropTypes.func,
+  selectedSource: PropTypes.object,
+  startPanelCollapsed: PropTypes.bool,
+  endPanelCollapsed: PropTypes.bool,
+};
+
+App.displayName = "App";
```

Comment:
Could we use a getter for displayName and propTypes?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get